web/src/components/List/ListItem/ListItem.scss

```diff
@@ -1,19 +1,21 @@
 @use "src/general.scss";
 
 .list-item-container {
-  display: grid;
-  grid-template-columns: 0fr 2fr 1fr;
-  grid-template-rows: 1fr;
```

Comment:
This was the root of the issue, the parent component having grid templating works in 99% of cases because 99% of cases also have image icons in the first column, in the case of the presets modal it does not and so it tries to cram the whole content into the first column

I've made it a toggleable argument option that is true by default to not have to change those 99% of cases